src/odemis/dataio/csv.py

```diff
@@ -62,13 +63,26 @@ def export(filename, data):
             csv_writer = csv.writer(fd)
             csv_writer.writerow(headers)
             csv_writer.writerows(spectrum_tuples)
-    elif data.ndim == 2 and all(s >= 2 for s in data.shape):
+    elif data.ndim == 2 and all(s >= 2 for s in data.shape) and (data.metadata.get(model.MD_DESCRIPTION, None) == "Angle-resolved"):
```

Comment:
That metadata wouldn't be set in the drivers (detectors), but in the streams (which is the first place where we know what acquisition type the data is for).
I'd suggest to first only use that for the raw export of Spectrum/AR. If next week we are still happy with this idea, we can go further with it.